### PR TITLE
Version 0.26.0

### PR DESCRIPTION
**Pull request by karpetrosyan**
### Added

* The `proxy` argument was added. You should use the `proxy` argument instead of the deprecated `proxies`, or use `mounts=` for more complex configurations. (#2879)

### Deprecated

* The `proxies` argument is now deprecated. It will still continue to work, but it will be removed in the future. (#2879)

### Fixed

* Fix cases of double escaping of URL path components. Allow / as a safe character in the query portion. https://github.com/encode/httpx/pull/2990.
* Handle `NO_PROXY` envvar cases when a fully qualified URL is supplied as the value. (#2741)
* Allow URLs where username or password contains unescaped '@'. (#2986)
* Ensure ASGI `raw_path` does not include URL query component. (#2999)
* Ensure `Response.iter_text()` cannot yield empty strings. (#2998)

### CODE REVIEW

**Comment by karpetrosyan** [comment]
Here is the release draft: https://github.com/encode/httpx/releases/tag/untagged-117424d33cff82b70e02

**Comment by T-256** [comment]
Don't forget https://github.com/encode/httpx/pull/2990

**Comment by tomchristie** [comment]
Thanks @T-256 - apologies for missing the `CHANGELOG.md` entry on that.

I'd suggest...

* Fix cases of double escaping of URL path components. Allow `/` as a safe character in the query portion. #2990.

**Comment by karpetrosyan** [comment]
Thanks @T-256 

I think this one is better

> Fix cases of double escaping of URL path components. Allow `/` as a safe character in the query portion. #2990.

**Comment by tomchristie** [comment]
> I think this one is better

Ok. Let's go with that.

**Comment by karpetrosyan** [comment]
Should I schedule it for today?

**Comment by tomchristie** [comment]
That would be fine, yep.
Thanks so much, @karpetrosyan. 😄

**Comment by karpetrosyan** [comment]
Here is the new one: https://github.com/encode/httpx/releases/edit/untagged-893baae112e60ca1d83c

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,14 +4,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## Unreleased
+## 0.26.0 (TODO)
```
Comment (karpetrosyan) [inline CHANGELOG.md:7]:
```suggestion
## 0.26.0 (20th December, 2023)
```

**Review by tomchristie — APPROVED** [review]
We might want to make it clear the `proxies=...` does still continue to work, but will raise a warning?

Looking great, thanks.

**Comment by karpetrosyan** [comment]
I'd recommend something like this.

## 0.26.0 (TODO)

### Added

* You should use the `proxy` argument instead of the deprecated `proxies`, or use 'mounts=' for more complex configurations. (#2879)

### Deprecated

* The `proxies` argument is now deprecated. It will still continue to work, but it will be removed in the future. (#2879)

**Comment by tomchristie** [comment]
Neat.

**Comment by karpetrosyan** [comment]
The final one: https://github.com/encode/httpx/releases/edit/untagged-8838a3cfc61a09bc35d6

**Comment by tomchristie** [comment]
Hrm... we don't seem to have triggered a `publish` event?

https://github.com/encode/httpx/actions/workflows/publish.yml

**Comment by tomchristie** [comment]
Ah okay, done.

**Pull request merged**